src/mame/video/k057714.h

```diff
@@ -39,6 +50,8 @@ class k057714_device : public device_t
 		VRAM_SIZE_HALF = 0x2000000 / 2
 	};
 
+	inline void crtc_set_screen_params();
```

Comment:
There’s no point putting the `inline` qualifier here when you’ve got it on the body.